src/pages/Organization/OrganizationUsers.tsx

```diff
@@ -150,11 +149,11 @@ export default function OrganizationUsers({ id, navOrganizationId }: Props) {
           />
         </div>
         {isFetchingUsers ? (
-          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4">
+          <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
             <CardGridSkeleton count={6} />
           </div>
         ) : (
-          <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4">
+          <div className="grid grid-cols-1 gap-4 lg:grid-cols-2 xl:grid-cols-3">
```

Comment:
152 and 156 should be the same 😅 

Anyways, since we are at it, let's move this up (before line 151, so we don't duplicate it).